### ISSUE TITLE
bug: core card: analytical card icon missing text alternative for screen reader and tooltip

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#3546]
#### Is this a bug, enhancement, or feature request?

bug

#### Briefly describe your proposal.

The status icon on the analytical card isn't read by the screen reader. Icons should have text alternatives for assistive technology. Icons and images should have a tooltip.
env: https://fundamental-ngx.netlify.app/#/core/card 

#### If this is a bug, please provide steps for reproducing it.

1. enable a screen reader (TTS)
2. navigate to the analytical card
3. click on the header and pay attention to the screen reader output (notice there is nothing relating to this icon)
3. click on the icon to elicit it specifically (no results)

![2020-10-05_3](https://user-images.githubusercontent.com/47522152/95053262-d92a0d80-06f8-11eb-8b07-f17596832250.png)
